server/config/config.go

```diff
@@ -313,10 +326,15 @@ func defineFlagsForMembers(parentStructNames []string, T reflect.Value) {
 		default:
 			// We know this is not flag compatible and it's here for
 			// long-term support reasons, so don't warn about it.
-			if fqFieldName != "auth.oauth_providers" {
-				log.Printf("Skipping flag: --%s, kind: %s", fqFieldName, f.Type().Kind())
+			if fqFieldName == "auth.oauth_providers" {
+				continue
 			}
-			continue
+			// Not supported via flags.
+			if fqFieldName == "storage.disk.partitions" || fqFieldName == "storage.disk.partition_mappings" ||
+				fqFieldName == "cache.disk.partitions" || fqFieldName == "cache.disk.partition_mappings" {
+				continue
+			}
```

Comment:
How do you propose we handle this once we transition to flags?

Reply:
Sent https://github.com/buildbuddy-io/buildbuddy/pull/707 to support slices of structs using json.